config/initializers/cf_client.rb

```diff
@@ -0,0 +1,12 @@
+require 'cf'
+
+Rails.configuration.api_endpoint = ENV.fetch("CF_API_ENDPOINT")
```

Comment:
I think this `Rails.configuration` entry would be better named `cf_api_endpoint` to avoid any possible ambiguity.